docs/contrib/PlatformBringUp.md

```diff
@@ -268,9 +265,8 @@ level-sensitive latch.
 
 This file is only required if you want to infer latches for your design.
 
-```
-// cells_latch.v
-//////////////////////////
+```{code-block} verilog
+:caption: cells_latch.v
```

Comment:
@luarss 
Does this work If the `:caption:` keyword is in the line above? Example
```
```{code-block} verilog :caption: cells_latch.v
```

Reply:
@vvbandeira  It doesn't work. Attached is the output for single line

![image](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/39641663/5c73d57e-92b3-49cf-9205-aea9e7e184bc)